LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/JoinRequestMessageHandler.cs

```diff
@@ -266,8 +266,8 @@ internal async Task ProcessJoinRequestAsync(LoRaRequest request)
 
                 var joinAcceptBytes = loRaPayloadJoinAccept.Serialize(appKey);
 
+                // For join accept messages the RX1DROffsetvalue is ignored as the join accept message carry the settings to the device.
```

Comment:
nit: is this relevant?

Reply:
Without this change I was not able to build locally, therefore I moved the comment before